http/client.go

```diff
@@ -479,6 +479,16 @@ func (c *Client) Purge(ctx context.Context) error {
 	return err
 }
 
+func (c *Client) HealthCheck(ctx context.Context) error {
```

Comment:
thought: This looks very similar to the previously removed `ping` command that we used to have.